podaac/umms_updater/umms_updater.py

```diff
@@ -183,10 +188,11 @@ def main(args):
         current_token = token_req.token(args.env, args.cmr_user, args.cmr_pass)
     else:
         current_token = args.token
+
     provider = args.provider
     header = {
         'Content-type': "application/vnd.nasa.cmr.umm+json;version=1.3.4",
-        'Echo-Token': str(current_token),
+        'Authorization': str(current_token),
```

Comment:
Did you test this? I think if in `Authorization` header it needs to be `f'Bearer {current_token}'`

Reply:
Yes, from what i saw in slack discussion launchpad tokens aren't oauth 2 tokens so the bearer headers don't work with cmr api. The authorization header works with the echo tokens and launchpad tokens.